salt/states/win_firewall.py

```diff
@@ -73,3 +93,54 @@ def add_rule(name, localport, protocol="tcp", action="allow", dir="in"):
         ret['comment'] = 'A rule with that name already exists'
 
     return ret
+
```

Comment:
Could you please add one additional line break here so that this matches our style guidelines? Thanks.